config/locales/views/about/hi.yml

```diff
@@ -0,0 +1,17 @@
+hi:
+  about:
+    title: "सर्किटवर्स - हमारे बारे में"
+    main_heading: "सर्किटवर्स के बारे में जाने"
+    main_description: "सर्किटवर्स के पीछे योगदानकर्ताओं के बारे में जानें"
+    product_details: "सर्किटवर्स आईआईआईटी-बैंगलोर के छात्रों द्वारा विकसित एक प्रोडक्ट है।
+          इसका उद्देश्य एक ऐसा मंच प्रदान करना है जहां ग्राफिकल यूजर इंटरफेस का उपयोग करके सर्किट की रचना और उसका सिमुलेशन किया जा सकता है।
+          जबकि उपयोगकर्ता सिम्युलेटर के द्वारा पूर्ण सीपीयू कार्यान्वयन की रचना कर सकते हैं, सॉफ्टवेयर की रचना प्राथमिक रूप से शैक्षिक उपयोग के लिए की गयी है।
```

Comment:
Here remove जबकि and add “lekin” before software.

Reply:
'lekin' means 'but' , the sentence ```while we can design cpu in simulator``` which represents a particular features. 

in my opinion using ```but we can design cpu in simulator``` in hindi  doesnt give any context of representation of features

above explanation sounds good?

Reply:
Yeah 👍🏻